extensions/wikia/GlobalNavigation/styles/GoogleSearchAPI.scss

```diff
@@ -0,0 +1,5 @@
+#___gcse_0 {
+  padding: 0;
+  height: 0;
```

Comment:
CSS declaration should be in alphabetical order. Consider using SCSSLint: https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md